bbl-viewer.appcache

```diff
@@ -0,0 +1,116 @@
+CACHE MANIFEST
+# 2018-08-21:v1
```

Comment:
Would it make sense to use the currently released version number (3.2.0) instead of `v1` here?

Reply:
I don't think so, except when you extend it with a build number or something similar - but we will have to check the standard if that's allowed. Otherwise it will get painful to test on a webserver as a browser will only then fetch new and changed files if the version number changes (you can see in the console when the browser starts checking and refreshing the files). I've interpreted it as the "version of the day" and incremented it accordingly. I did not dig the cache manifest standard on the correct use. We probably should do that to understand how we have to use it.